

\documentclass[11pt,letterpaper]{article}
\usepackage[a4paper,width=150mm,top=25mm,bottom=25mm,bindingoffset=6mm]{geometry}
\usepackage{natbib}
\bibliographystyle{plainnat}
%\bibliographystyle{plos2015}

%\usepackge{fancyhdr}
%\pagestyle{fancy}

\usepackage[utf8]{inputenc}
\usepackage{graphicx}
\graphicspath{ {images/} }

\usepackage[left]{lineno}
\usepackage[table]{xcolor}

\usepackage{setspace} 
\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

\usepackage{microtype}
\DisableLigatures[f]{encoding = *, family = * }



\begin{document}

%begin{titlepage}
    \thispagestyle{empty}

    \begin{left}

    \includegraphics[width=0.4\textwidth]{imperial_logo.eps}
    
    \end{left}
    
    \begin{center}    

        \vspace{1.5cm}

        \LARGE
        \textbf{How does adding data change node age estimation in morphological phylogenies?}
        
        %\vspace{0.5cm}
        %\LARGE
        %Thesis Subtitle
        
        \vspace{3.5cm}
        
        \Large
        \textbf{Joseph Craig\\
        CID:01282008}
         
        \vspace{1.5cm}
         
        \large
        A thesis submitted in partial fulfilment of the requirements for the degree of Master of Science at Imperial College London \\
        Formatted in the journal style of the !!!! jounal\\ %TG PLoS ONE
        Submitted for the MSc in 2017\\
        
        \vspace{1.0cm}
                
        \large
        Imperial College London\\
        September 2017\\

        \vspace{1.0cm}

        \large
        Supervisor: Dr. Martin D. Brazeau \\ %TG: The otherway around no? I'm not sure even if Martin needs to be there as a supervisor.
        Co-supervisor: Dr. Thomas Guillerme \\
        
    \end{center}
%\end{titlepage}

\newpage

\section*{Declaration}

The following investigation was undertaken....

The data used was assembled from available sets in previously published papers.

The data was processed by myself.

Any novel functions or alterations to existing functions were written by Dr. Thomas Guillerme.


\newpage

%\section{Abstract}
\setcounter{page}{1}
\thispagestyle{plain}
\begin{center}
	\large
	\textbf{How does adding data change node age estimation in morphological phylogenies?}

	\vspace{0.4cm}
	\textbf{Joseph Craig}

	\vspace{0.9cm}
	\textbf{Abstract}
\end{center}

This aim of this investigation was ...
This is becuase ...
This was achieved by ...
What we found was ... 
This is exciting becuase ...
This suggests that/in the future ...

\newpage

\tableofcontents

\newpage
% \linenumbers

\section{Introduction}

%TG: General note, especially for the method part. Write in active form! Never ever use passive form in papers. It's pedantic, boring and generates more complex sentences. "Trees were infered using a method" Should be written "We (I) infered trees using a method".

Phylogenetics has been a fundamental cornerstone of evolutionary biology since Darwins first drawing in The Origin of Species, %TG: technically this is wrong. Phylogeny became only important later and were already drawn before Darwin. But no big deal: just try to avoid these "Since Darwin" sentences, they don't bring much and sound awfully "undergraduate" style.
 helping to elucidate the relationships and the divergences between different taxa.
 Moreover, these divergences can be estimated in relation to time, allowing us to propose exactly %TG: or more "propose with some narrow error" rather than exactly
  when speciation events may have occured.
The timing of these pivitol events can be of great importance when studying evolutionary history and many advances have been made to the techniques used in order to more precisely estimate these significant evolutionary dates. %TG: this sentence needs rewriting. I get your idea but it's writen a bit clumsily. Try something like: "Timing of speciations are important to know [some example] and there has been many novel and improved techniques to get this dates with more and more accuracy."

Billions of species have existed on earth during its history, the majority of those species are now extinct and therefore a large proportion of lineages no longer have any extant taxa through which molecular data can be gained and analysed using classic clock methods \citep{turner2017empirical}. %TG: you need to introduce molecular data a bit more carefully. Maybe something like "To look at this relations we can use molecular data. However, a lot of species are extinct blablabla".
To successfully date these phylogenies requires the use of morphological data from fossils.
The chronostratigraphic %TG: add in brackets what it means (i.e. the age of the fossils)
data available from fossils is the only explicit source for absolute timing in phylogenies \citep{turner2017empirical}, making them indispensable in analyses, despite some drawbacks.
The incorporation of these data comes with uncertainty from missing data, difficulties in correctly dating stratigraphies, identifying specimens and the ever present knowledge that each specimen is only the earliest recorded appearance of a species and due to variation in preservation and perhaps not its true origin. %TG: the last part "the ever present knowledge that..." should be explained a bit more carefully. I guess you are talking about fossilisation bias right?
Using fossils to date phylogenies started with their application to internal nodes using prior probability distributions. %TG: not really. fossils were used since way longer than phylogenetics. I think you mean that "We started to use fossil into dating molecular phylogenies by using their age to calibrate nodes in the phylogenies (e.g. by using the fossil age as a prior for the node age)."
An early method to dating the origins of phylogenetic phenomona was called ghost lineage analysis, named for its side effect of predicting unsampled fossil diversity \citep{norell1992taxic}. %TG: There's a confusion here. Ghost lineages is not a phylogenetic method at all! Ghost lineages appear (sometimes) when you don't calibrate a node, this node is aged more recent than the fossils we know associated to this group and that implies that for the period of time between the fossil age and the node age, we have no traces of this lineage in the fossil record (hence the ghost). Also the start of your sentence is not really nice.
This method took the stratigraphic age ranges of fossils and connected them to undated trees produced by cladistic parsimony. %TG: again, this is incorrect. 1 - Cladistic is the science of generating clades, it can be done via parsimony, bayesian, or anything else. 2 - even though some people fixed the topology based on a parsimony tree for then dating the tree, this is not at all a generality. Node dating can totally estimate both the topology and the node age.
Then on the assumption that sister lineages will have the same origin date with more distant related species branching earlier, ghost lineage analysis could use the oldest member of each sister clade to predict the minimum divergence dates \citep{norell1992taxic}. %TG: again, this is not correct
This method however will always only provide the minimum time estimate, and cannot describe the true date of origin. %TG: same here. Have a look at Bromham and Penny 2003 (that I'll attach on Slack)

The earliest method for dating phylogenies using more than just fossils was achieved by assuming a constant molecular clock that was calibrated using the fossil record \citep{zuckerkandl1962molecular}. %TG: ah OK... I kind of see what you mean in the section above: you're talking about strato-cladistics and now about molecular dating. Frankly, since you're not doing any stratocladistics, you can start directly from this paragraph with Zuckerkandl's first molecular clock. In general be carefull with these "history" parts in an intro. You don't want to write a review on dating in phylogenetics (though it would be interesting) but just introduce the reader to the methods and problematics you are going to work on. This can be done by a simpler background. Remember pro-tip: more clarity = less text = less mistakes.
However it soon became apparent that this constant molecular clock was not adequate as over time and across lineages there was a significant variation in the rate of molecular evolution.
This led to the use of the relaxed-clock models that had multiple points for calibration from fossils \citep{drummond2006relaxed}.
When paired with models of speciation event distribution over time such as; the Yule model, that assumes a constant rate of speciation across lineages with no extinction, \citep{yule1925mathematical} or the birth-death process, in which a lineage at any point in time can speciate or go extinct and connected by diversification rate and turnover \citep{kendall1948generalized} we can infer the absolute rates and node ages.
To accommodate this developing complexity in the methodology and to cope with the increasing size of data sets, analyses using Bayesian inference has also become more prevelant, allowing uncertainty in analyses to remain.
Despite these considerations on rate variation, there has been little change to the calibrations themselves.
A relatively new method for calibrating phylogenies is the fossilized birth-death process \citep{heath2014fossilized}, a prior for dating divergence times, which aims to define the distribution of fossils and speciation times.
This is acheived by informing on the uncertainty for each speciation event in a phylogeny through its parameters: Speciation rate, extinction rate, fossil recovery rate and the proportion of sampled extant species.%TG: two things: 1- I think this fossilized birth-death model can be explained a bit more carefully and 2 - the fossilized birth-death is no a calibration method. Basically you have two aspects to phylogenies: the topology (relations between sp) and the branch length (distance between sp). The classic birth-death model calculates branch length by estimating speciation and extinction rates (and the turnover which is speciation/extinction) to create realistic trees (cf the Yule that is based on random branch length). The fossilised birth-death is just a clever improvement that corrects speciation and exctinction estimations by adding a fossilisation component (i.e. what is the change of recovering a speciation/extinction knowing the fossil record).

Although node dating is the most common method of phylogenetic dating with over 2000 references, it requires indirect methods that may lead to misrepresentation of the data \citep{turner2017empirical}. %TG: you will need to quickly develop what misinterpretation maybe something like: "... may lead to misrepresentation of the data (e.g. [some example], Turner et al 2017)".
 However, this is not the only available avenue for the information fossils provide. %TG: rewrite this sentence
In paleontological data sets containing some extant species then tip-dating can provide divergence estimation along with absolute rates and times when coupled with the models of evolutionary change \citep{pyron2011divergence}. %TG: I am not sure what you're trying to say in this sentence. 
%TG: There is a slight misunderstanding with the tip-dating/Total-Evidence method below. Total Evidence is easy: it's just combining morphological and molecular data in a matrix (no dating, etc...). We're not doing this here. Tip-dating is, as you got it using fossils as tips rather than nodes. This doesn't mean that node-dating (using fossils as nodes) ignored the fossils, it just made the assuption that the fossil you're using to calibrate a node is de facto the ancestor of the descendants of the node (and can then be used to determine when the to descendants split). The tip-dating doesn't make assumptions on the node's age using the fossil but rather uses fossils as tips (not nodes) and estimates ages from there. Have a look at O'Reilly and Donoghue 2016 for more info (attached in Slack).
Tip dating allows fossils to be fully intergrated within phylogenetic analyses tree building, rather than provisionally. Recently this methodology has been adapted and applied to morphology-based fossil-only datasets \citep{slater2015iterative}, such as the data set used in our study. %TG: remove the part concerning the study, you didin't introduced that yet.
Total-evidence data takes this one step further by using this paleontological data to derive the placement of fossils at the same time as calibrating the tree, thereby removing the need to directly calibrate or place fossils onto the tree. This can improve the resolution and provide greater support for ages and posterior probabilities whilst being less sensitive to priors.  

Time-calibration is a fundermental step in phylogentics that allows for a multitude of evolutionary questions to be asked, from patterns of coevolution and speciation to historical biogeography and climatic-biotic interactions over time \citep{heath2014fossilized}. One such date of significance with particular popularity and contention is the origin of placental mammals that has been an ongoing debate for decades. For a long time the most popular theory was that the `Age of Mammals' began with the Cretaceous-Paleogene mass extinction that brought the non-avian dinosaurs to an end and allowed the rapid radiation of mammals. %TG: be careful with all the literacy license (e.g. "brought the dinosaurs to an end"). For some reviewers (e.g. me) it can be really annoying. This is science writing, not litterature writing. As I told you, it's OK when you're a good writer (I'm not judging on that here) but becomes plainly boring when not.
Whilst a post K-pg date for thier radiation is still widely accepted,%TG: not at all! You can read these two opinion pieces I wrote a while ago to get a gist of the debate: http://www.ecoevoblog.com/2013/02/18/palaeo-poetry-and-placental-mammals/ and http://www.ecoevoblog.com/2013/10/18/the-placental-mammal-saga-special-summer-double-episode/ . Basically the debate is between three competing hyothesis: the "long-fuse" (placental mammals appear and diversify long before the K-Pg), the "short-fuse" (placental mammals appear longish before K-Pg but only diversify only around or after K-Pg) and the "burst/explosive" (placental mammals appear and diversify after the K-Pg).
 the exact date of thier origin is much more debated. There is evidence for the early Cenozoic \citep{wible2007cretaceous} dated by the oldest definitive paleontogloical evidence for placentals, through the late Cretaceous \citep{archibald2011protungulatum}, \citep{hooker2014new} and even earlier dates brought to light through molecular dating estimates \citep{dos2012phylogenomic}.   

%TG: you need to spend more time explaining the placental debate, what causes it and why it's important (and maybe less time or non at all on the strato-cladistics).

The main objective of our study is to illustrate how node age estimations can change with the addition of data when using morphology-based fossil-only data sets. %TG: there are actually a couple of living taxa in the data set.
 Futhermore additional aims seek to experiment with the new parameter, the fossilized birth-death process. This will be achieved by comparing a new tree, created from the data sets of two recently published trees aimed to date the origins of placentalia, and those original trees.
 The new tree will be built using Bayesian inference and the fossilzed birth-death process. %TG: this goes in the methods
 The first original tree published by Beck and Lee (2014) was also built using Bayesian inference and a relaxed clock method. The second original tree published by Halliday and Goswami (2015) was built using Maximum Parsimony. %TG: this to

 %TG: this last paragraphe needs rewriting: why we chose to look at that? How is it important, what are we going to do in the great lines (i.e. compare node ages) and what did we found (in the great lines too).

\newpage

\section{Methods}

%TG: you need some context here. From what I read above, you just detailed some bits of methods in phylogenetic inference but we still have no clue where you're leading us. The methods must come as a logical decision to your research question in the introduction. For example why do you need to combine the matrices, create the trees and estimate node age? How does that fit in the narative you introduced above? Also for the part below, use the proper citation format (like in the introduction) rather than writing down the citations each time.

\subsection{Data Combination}

%TG: start with: "We first combined a matrix blablabla."
The final combined matrix used in this analysis consists of 221 %TG: way more no?
 taxa from across the Eutharian clade with 6 outgroup taxa %TG: I thought we used only Nanolesthes? Also, you're talking about placental mammals above and now about Eutherians, you need to define the differences.
and 694 characters (dental, cranial and postcranial), creating an extensive morphological data set of Eutharia and its outgroups.
The taxa and characters used were collected from two discrete morphological data matrices and compiled into a super matrix with a standardised formatting and naming system.
The first of these data sets was taken from Beck and Lee's (2014) and this provided 421 characters, 13 of which were unique between the two data sets, with 102 taxa, 47 of which these were unique.
The second data set was taken from Halliday and Goswami's (2015) !!!! which provided an additional 222 novel characters and 680 characters in total with 177 taxa, 115 of these were unique.
%TG: it would be good to have a table representing all that. Somthing like this:
% \begin{table}
% \center
%     \begin{tabular}{r|cccc}
%             & Taxa & Characters & Unique taxa & Unique characters\\
%         \hline
%         \textit{Beck and Lee 2014} & 1 & 1 & 1 & 1\\
%         \textit{Halliday and Goswami 2015} & 1 & 1 & 1 & 1\\
%         \textit{This study} & 1 & 1 & 0 & 0\\
%     \end{tabular}
%     \caption{...}
% \end{table}

To successfully combine these two matrices, they were both formatted %TG: not the matrices, the characters
 with the following system: the `Author' %TG: why quotes?
 , referring to the oldest known use of that character, the `Original character number' within its first publicised use where available %TG: or more: that was its position in the first publicised matrix or something like that?
 , their `Name' and their states.
 The `Name' given to each character was created using the first 3 letters of the author, the date of its original publication final 2 digits of the year, the character type code full list in supp !!!! and the numerical identifier within each type. %TG: it will be good to add an example here: "for example character Bec14.01 is for...".

The states of each character where kept as originally described with the exception of characters shared between the matrices with the description by Halliday selected as the most accurate or up-to-date. Some of the original characters from Beck and Lee's (2014) matrix were split into several more specific sub-characters in Halliday and Goswami's matrix (2015); these were grouped together and preserved during the combining of the matrix. %TG: add this information to the supplementary

To create the combined matrix used in this analysis we used R v. 3.4.1 R Core Team, (2014), the full code is provided in the supplementary data.
The origianl character data sets from Beck %TG: it should be Beck and Lee (2014) throughout, same for Halliday
and Halliday were imported to R using a modified ``ReadMorphNexus'' from the Claddis package, allowing ... %TG: !!!!
From this data the matrices were extracted and those species that were common to both were matched to prevent over-representation of individual species.
We used Halliday's matrix as our starting point as it was the larger of the two, then in order to create the combined matrix we first had to create sub-matrices.
The first of these contained Beck's unique species with Halliday's characters, using a loop to correctly place the characters in their columns this sub-matrix could be added to Halliday's.
The second sub-matrix consisted of Becks unique characters and Halliday's taxa; this was then combined with a third sub-matrix made up of Beck's taxa and unique characters.
Both were then added to the main matrix to create the final combined matrix.
Some shared characters had states that had been reversed or dis-ordered between the two orignal matrices.
Therefore these required converting within the final matrix using a conversion list and created function `flip.characters'.
%TG: I'm not sure if that part is not too detailed. Maybe you'll want to reduce this a bit (especially about the name of the functions...). Instead you can insist on the amount of work you did there: you went through 660+ character definitions and checked wchich one matche which one, etc.

\newpage

\subsection{Tree Inference}

We created our trees using bayseian inference in the MrBayes software cite!!!!.
Due to shortages in available time, both trees were created from 2 runs with 4 chains and were run with a priori constraints in keeping with current views on placental phylogeny and therefore avoid much noise from the unconstrained topologies and speed up the mcmcs. %TG: MCMC is capitalised
These constraints were created using the `paleotree' package in R and using `ape::consensus`.
For the Halliday constraint, we built a majority consensus tree from the topologies of Halliday and Goswami (2015) http://onlinelibrary.wiley.com/doi/10.1111/bij.12731/abstract %TG: remove the link, same below
without branch lengths.
For the Beck constraint, we built a majority consensus tree from the 7 constrained topologies of Beck and Lee (2014) http://rspb.royalsocietypublishing.org/content/281/1793/20141278 without branch lengths.
For our outgroup we used Nanolestes, %TG: species/genus names in italic please (throughout the text)!
 the oldest non-therian cladotherian taxa. %TG: define these terms (cladotherian, etc.) if you use them. Even I only have a vague clue of what they mean...
 We constrained the ingroup and root to seperate groups and calibrated the root with a uniform date of between 161-199 mya, %TG: mya is an acronym (usually Mya), define it the first time. E.g. "161 to 199 million years ago (Mya)"
  mid-late Jurassic.
 Independent gamma rate clock was used for the rate at which the variance of the effective branch length increases over time. %TG: this doesn't mean much! The rate at which the rate of the (effective?) branch length increases? Be really careful with those, it's easy enough for a reader to spot that you didn't understood what you wrote here and that you're making up by using "complex" terminology. For the reviewer it's going to be annoying and will not be at your advantage.
 Fixed tip dates were added to further constrain the topology of the trees and speed up the mcmc. %TG: no the tip dates are not for speeding up for for calculating brranch length using tip-dating.
 The dates were taken from Beck and Lee's (2014) parameters for their taxa, Hallidays additional taxa were dated using the oldest dates available for each taxon on fossilworks. %TG: This is not clear. I know what you've done but I still needed to read this sentence twice. pro-tip, read these sentence aloud (preferably to a colleague) and see if they make sense! Also, you can simplify by saying you got all the dates from the palaeo data base (or fossilworks, it's the same) since that's where Beck got them from.
 For the two species Eomanis and Eurotamandua whose exact/direct ages were not available on fossilworks, we used ages from that of the messel pit formation where both species were discovered.
 The clock model chosen for this analysis was a relatively new method of fossil calibration. %TG: No. See my comments throughout the text.
 The aim of the fossilized birth-death process is to more accurately estimate node ages by clearly stating that extant taxa and fossil taxa are under the same evolutionary process, calibrating nodes through an individual diversification model. %TG: same.


%TG: on a general note for the paragraph here, I would restructure it into something more classic: first the things in common (which datasets, which models, which dates, which mcmc parameters) and then what differs (which constraint and why).

\newpage

\subsection{Node Estimation}

Node ages were estimated using R v. 3.4.1 R Core Team, with the ape and phylotate packages. %TG: No. ages were estimated in MrBayes, we just used R to extract these ages.
Two trees from Beck and Lee (2014) were used, the first with placentalia constrained to 98mya and the second with placentalia constrained to 66mya.
Two trees from Halliday were also used %TG: which ones?
and finally two trees from this study produced as above.
The trees were read into R where the mean, median and hpd 95\% values were extracted from the nodes of interest: Eutharia, Placentalia, Metatheria, Artiodactyla, Primates, Carnivora, Sirenia, Glires, Xenarthra, Paenungulata, Notoungulata, Euarchonta, Ferae.
The resulting data was summarised and is shown below. %TG: remove this sentence (this is expected anyways). I think you should more focus on this paragraph since it's the key answer to you question: actually analysing the differences in node ages.

\newpage

\section{Results} 

\subsection{Plot of node ages against data sets}

%\begin{figure}[!htbp]
%\centering
%   \includegraphics[width=1\textwidth]{path_to_your_figure.pdf} 
%\caption{blbalbalbal caption}
%\label{your_figure_label}
%\end{figure}

\subsection{Table of raw node ages}

%\begin{figure}[!htbp]
%\centering
%   \includegraphics[width=1\textwidth]{path_to_your_figure.pdf} 
%\caption{blbalbalbal caption}
%\label{your_figure_label}
%\end{figure}

\newpage

\section{Discussion}

We investigated the effect of adding data on node date estimation on large morphology-based fossil-only data sets.

The result of this study suggests...

We concluded that ...

Further research may include ... Could focus on... expand on...

Should be using molecular \citep{puttick2016dating} %TG: good point!

The main limiting factor of this study was that of available time and computational power. %TG: to be frank, as that was certainly a limitation, I'm not sure if it' the main. It's just time consuming to do these projects.
These limitations meant that our MCMC chains could not run for as long as we hoped or needed in order to fully realise the aims of the investigation. %TG: As I told you, it's normal to have these limitations, where your report will stand out as a good one is if you justify what was expected and what could have been done. Here you say the "MCMC could not run for as long as hoped" what did we hoped (the convergence)? How, if you'd rerun the project could you fix that? This should be discussed for each caveat.

Moreover heavy constraints were required to speed up the MCMC and only a few parameter combinations could be tested before running the final set.
With more time and resources it would be possible to fully explore more parameter options and compare a larger number of trees using the combined data set.
These would include running a maximum parsimony for the combined tree and Beck and Lee's (2014) original tree in order to allow better comparisons of both to Halliday and Goswami's (2015) original tree. %TG: why this parameter? Which other ones?
Additionally, further study could look at the effects seen here when the use of molecular data for the extant taxa was included.
The use of soley morphological data to estimate node ages, as we did here in the combined matrix tree, undoubtably warrants further study as this approach remains rather novel and the effects when paired with different models and methods undertested Beck and Lee, 2014 ;  Lee et al., 2014. 

\newpage

%\nolinenumbers

\section{Supporting Information}

\paragraph*{Combined Tree File}
\label{S1_Fig}
{\bf Bold the title sentence.} Add descriptive text after the title of the item optional.

\paragraph*{Original Halliday Tree File}
\label{S2_Fig}
{\bf Lorem ipsum.} Maecenas 

\paragraph*{Original Beck Tree File}
\label{S1_File}
{\bf Lorem ipsum.}  Maecenas 
\paragraph*{MB command file}
\label{S1_Video}
{\bf Lorem ipsum.}  Maecenas 

\paragraph*{R code}
\label{S1_Appendix}
{\bf Lorem ipsum.} Maecenas 

\newpage

\subsection{Other/Research/Ideas}


Maximum parsimony analysis works well for generating trees, it is less well suited at evaluating support for different phylogenetic relationships particularly with fossil taxa. Here, bayesian inference methods provide provide a more open and programmable approach, allowing a priori parameters to be applied. This is of particular interest here as bayesian inference allows a priori calibration of geological dates. A recent simulation study revealed that a Bayesian inference method outperforms Maximum parsimony when applied to discrete characters and when there is missing data.

Bayesian handles missing data better.

MCMC method estimates the posterior probability of a tree as its frequency in a distribution of trees. Trees are evaluated and retained in this distribution in an iterative manner, retaining trees with a probabiltity proportional to how much worse or better the new tree is compared to the last. Each new tree is a generation in a chain.

Dated phylogenetic analysis, a recent development in cladisitcs, takes advantage of fossil data and the geological associated with them to constrain branch lengths.   

MCC The trees sampled in a dated Bayesian analysis are summarized in a maximum clade credibility MCC tree, which shows the posterior probability for each clade. A clade's posterior probability is just the proportion of times the clade appears in trees in the MCMC sample. This means that the posterior probabilities of the MCC tree are coarse analogues of bootstrap values in the maximum parsimony framework.

A Bayes factor is a ratio of marginal likelihoods of two different hypotheses, and is interpreted as the relative ability of each hypothesis to predict the data. A hypothesis that is more likely to lead to the observed dataset will produce a higher marginal likelihood than one that is less likely to have given rise to the observed dataset, and this will result in a high Bayes factor. In Bayes factor tests, competing hypotheses are represented as differing topological constraints on trees, and the aforementioned procedure for inferring trees is employed to produce the best fully defined hypotheses of relationship consistent with each constraint. The fits of these constrained trees to the data can then be compared.

byproduct of dated Bayesian analysis is a “morphological clock” estimate of the age of any undated terminal node. Such ages are produced by combining the rate of change for each character derived from the underlying model, the inferred tree, and the ages of geologically-dated terminal tips on the one hand with the character states exhibited by the undated tip on the other. While morphological data have been used in concert with molecular data to estimate the age of taxa without geological dates in a number of dated Bayesian analyses e.g., Pyron, 2011; Ronquist et al., 2012a ;  Wood et al., 2013, 

------ we believe the exclusive use of morphological data to generate age estimates is novel, and remains untested see discussion in Beck and Lee, 2014 ;  Lee et al., 2014. With this in mind, we used a jackknife resampling procedure to assess the reliability of the morphological clock age for the Dinaledi fossils yielded by the dated Bayesian analysis. ----------- Possilbe further study discussion????

Bayesian inference of phylogeny uses a liklihood function to create a quantity called the posterior probability of trees using a model of evolution, based on some prior probabilities, producing the most likely phylogenetic tree for thte given data. The proir probability of a tree represents the probabiltiy of the tree before the observations have been made. Typically all trees are considered equally probable, a priori. However, other information can be used to give some trees more prior probability eg. the taxonomy of a group. The liklihood is proportional to the probability of the observations conditional on the tree. This probability requires making specific assumptions about the processes generating the observations. The posterior probability of a tree is the probability of the tree conditional on the observations. It is obtained by combinging the prior and liklihood for each tree using Bayes' formula.Parameter space is assessed the a Markov Chain Monte Carlo investigating new trees original trees are perturbed and assessed for their posterior probability and then either rejected or accepted. There is no endpoint optimum, only when the investigator ends the chain generations. A majority rule consensus tree is then presented.

Ideally, molecular clock calibrations are obtained from accurately dated fossils that can be assigned to nodes with high phylogenetic precision Graur and Martin 2004, but reality is generally far from this ideal because of a number of important problems. The incomplete and imperfect nature of the fossil record means that fossils necessarily only provide evidence for the minimum age of a clade. Many clades will be considerably older than the oldest known fossil; thus, nodes may be constrained to erroneously young ages Benton and Ayala 2003, Donoghue and Benton 2007, Marshall 2008. Incorrect fossil dates also arise from experimental errors in radiometric dating of fossil-bearing rocks or incorrectly assigning fossils to a specific stratum. In addition, misinterpreted character state changes can result in the taxonomic misidentification of fossils or their incorrect placement on the phylogeny Lee 1999. 

\citep{lee2015morphological}

However, with the recognition that fossil species are integral to correctly inferring patterns of character evolution and changes in diversity, as well as in establishing evolutionary timescales, morphological data are enjoying a phylogenetic renaissance 

\citep{dembo2016evolutionary}, \citep{o2016bayesian}


\section{Possible Papers/References}

\citep{halliday2016eutherian}
\citep{halliday2016impact}
\citep{halliday2017resolving}
\citep{halliday2016eutherians}
\citep{beck2014ancient}
\citep{arcila2015evaluation}
\citep{near2004assessing}
\citep{lee2015morphological}
\citep{puttick2016dating}
\citep{pyron2016novel}
\citep{turner2017empirical}
\citep{heath2014fossilized}
\citep{wiens2004role}
\citep{nabhan2011impact}
\citep{dembo2016evolutionary}
\citep{o2016bayesian}


\bibliographystyle{plainnat}
\bibliography{References/Biblo}

\end{document}

- Broad = what is the paper about and why is this interesting, why isn't there an answer already?\\
- Explanation of difficulties of phylogenetics as a broad topic.\\
- Accuracy of phylogenetics = consistency convergence on correct tree, efficiency rate of convergence with more data added, robustness how sensitive to violations of robustness\\
%TG: I'm not sure these three key words consistency, efficiency and robustness have the same meaning to what's described in brackets. Maybe prefer using your own words.
- Use of fossils for dating trees.\\ \citep{}
- Importance of node dating and divergence times. \citep{turner2017empirical} 
%TG: This whole paragraph seems a bit confusing, what are you actually trying to say? Where do you want to go with that? Regarding your title I would go more directly into the topic:
%TG: 1 - dates on a phylogeny or dating depens directly on which data is available and how much of this data is available. 
%TG: 2 - the more ancient fossils you have, the better the dates would be technically 

\subsection{Paragraph 2}
- Focusing
- Mammals in phylogenetics.\\
- Mammal divergence estimates around K-pg.\\
- 
%TG: develop more on the state-of-the-art in mammal phylogenies here: is there a consensus? What are the variations? Are they due to different methods or different datasets? What does these differences imply in our understanding of mammal evolution

% 

\subsection{Paragraph 3}
- Focused
- Hypotheses
- How is this going to be tested? 
- Why are we testing it in this way?
%TG: You can always end here with a brief sentence of "what we found".

\subsection{Other Ideas}
%TG: I am not going to much through these since they look more like lecture notes than part of the plan.
- define Phylogenetics\\
major Methods of phlyogenetics:\\ 
- Phenetic distance based methods - discrete morphological data\\ \citep{wright2014bayesian} - Bayes vs ML outperforms parsimony for phylogeny estimation from discrete morphological data. Bayes deals with uncertainty in a more  explicit way then ML, retaining the uncertainty of branch lengths, model parameters and other parameters in the results.

what can Phylogenetics do for us?\\
%TG: bit of confusion between the major methods here. NJ is an algorithm used in both Bayesian, MP and ML. Phenetic = related to the phenotype. Character in your case discrete morphological characters are totally phenetic. Bayesian and ML involve distance methods.

accuracy of Phylogenitic trees?\\
- consistency convergence on correct tree\\
- efficiency rate of convergence with more data added\\
- robustness how sensitive to violations of robustness\\
%TG: same comment as above. These "buzz" words mean other things than your definition in brackets.

missing data effects\\

molecular improvements\\

models\\

fossils in phlyogenetics - 50 billion species, 99\% extinct \\
%TG: where did you got that 50 Billion value???
- fossils can provide a strong basis of evolutionary classification, when adequate data is available \\
- fossils can alter exisiting relationships and infer new ones\\
- Break-up long branches\\
- Stabalise uncertain polarities\\
- Sequence events\\
- Calculating rates\\
- minimum ages and the appearences of extant life. Fossils = only tangible evidence of ancient life.

fossil calibrations\\
- node dating\\
- tip dating \\
- total evidence analysis node + tip dating\\
%TG: No Total Evidence = molecular + morphological data. Nothing to do with dating.

Node dating = most common method >2000 references\\
Node and tip dating = improves the resolution and greater support for ages/posteriors, can be less sensitive to priors\\

Yule process = assumes a constant rate of speciation across lineages no extinction.\\

Birth-death process = at any point in time a lineage can speciate at rate λ speciation rate or go extinct with a rate of μ extinction rate, connected by diversification rate and turnover.\\

Fossilized birth-death process = probability of the tree and fossil observations under a birth-death model. Improves inference of nodes. Better capture statistical uncertainty. Turnover and diversification rates now linked by fossil recovery rate and the probability of fossil recovery.\\

- What’s the main thing we know now that we didn’t know before? What affect did adding data have on the node ages?\\

- What’s the chain of logic and results that means we know this answer?\\ %TG: uh?

- How does this affect our future research? What are the possible implications of this answer?\\

- What future work could build more broadly on what we’ve found?\\
- How may this study in this area be of interest to other work in other areas, or other systems.\\

- Link\\

1. Gather data in the form of two morphological data sets \\
 - Beck and Lee character matrix \citep{beck2014ancient} 102 taxa 47 unique, 421 characters 13 unique combined with..\\  
 - Halliday and Goswami matrix \citep{halliday2016eutherian} 177 taxa 115 unique, 680 characters 222 unique.\\ 
 - 59 common species\\

2. Create a super matrix in excel to standardise the data format and naming system.\\ %TG: the fact that you're using Excel is not important here.
 - Characters formatted as: \\
 - Author = Original author of character.\\
 - OCN = Original character number where found.\\ %TG: try to avoid abbreviations like plague. Some are pretty usual like ML - though still needs to be defined but making up some makes the text just more annoying to read and understand I don't know if you where planning to use them.
 - Name = First three letters of author, date of publication last two digits of four digit year, code of character type, number identifier within.\\ character type. \\ 
 - During creation of the super matrix, some descriptions of shared characters were different. In these cases Hallidays descriptions were taken.\\
 - Some of Becks original characters were broken %TG: or "split"?
  down into subs, more specific characters, by Halliday.\\

3. R was used to combine these two data matrices full code in supplementary?.\\ %TG: yes, good idea for the code!
 - The character data from Beck and Halliday was inported to R using a modified ``ReadMorphNexus'' from the Claddis package.\\
 - The matrices were extracted and combined based on unique and common taxa.\\
 - A loop was written to place the added characters into the correct position in the super matrix.\\
 - Some shared characters had flipped states in Hallidays data, a loop was used and a function written ``flip.characters.R'' to re-order the states of these characters according to a list.\\
 %TG: If you're adding the code in the supplementary, which function you use is not really important. Maybe focus on describing what the code does rather than which one.

 4. A modified ``write.nexus.data'' function was used to export the finalised matrix.\\
 %TG: same, focus on what it's doing.

 5. Paleotree package was used to create the two constraints used. After %TG: make sure you write the package names correctly paleotree has no capital P
 - Halliday's topology constraint\\
For the Halliday's constraint, we build a majority consensus tree from the topologies of [Halliday and Goswami 2015]\\ http://onlinelibrary.wiley.com/doi/10.1111/bij.12731/abstract without branch length using `ape::consensus` in R.
 - Beck's topology constraint\\
For the Beck's constraint, we build a majority consensus tree from the 7 constrained topologies of [Beck and Lee 2014]\\ http://rspb.royalsocietypublishing.org/content/281/1793/20141278 without branch length using `ape::consensus` in R.
We selected the constrained trees only to avoid to much noise from the unconstrained topologies.
%TG: not sure about the noise. The constrain is more to speed up the MCMC.

1. Trees were created using Mr Bayes Nexus files available in Supp.\\

2. parameters\\
- outgroup Nanolestes = why? %TG: yes you need to specify that
- delete taxa Laindon, Montamalestes, Kharmerungulatum, Alymlestes = Because these are unstable taxa\\
- constrained the ingroup and root to seperate groups 1 and 2\\ %TG: why?
- clockvarpr=igr independent gamma rates =it is the rate at which the variance of the effective branch length increases over time,\\
- brlenspr=clock:fossilization birth-death-rate = why?\\ %TG: yes, you need to explain
- fixed tip dating = Beck species tips dated as Becks dates, Hallidays additional species dated using fossilworks, for these two species !!!! ages were taken from the age of the messel pit formation. = why?\\
- constrain the combined tree by Becks original tree, then Hallidays original tree, [then by both combined !!!!]\\
- 2 runs = why?\\
- 4 chains = why?\\
- burnin of !!!! = why?\\
%TG: the last three parameters don't require that much justification. It is important however to explain the other bits. They are advanced phylogenetic methods and the people that will mark your report will want to know if you just used them because I said so or if you actually understand what's happening there.

%TG: where does that section fit?

combine two morphological data sets using R \\
- Assumptions: \\ 
- Halliday character states/descriptiions updated Becks\\ 

- Alterations:\\
- Some characters from Beck and Lee were broken down in Halliday and Goswami’s matrix. \\
- Some character states were also reversed between matrices. These were altered in the nexus and are marked in the character table. \\
%TG: this needs to be explained
 
Combined using R - Adding Becks unique taxa and characters to Hallidays matrix. \\ 

use composite %TG: or combined?
 matrix to run Bayesian analysis\\
create constraints using PaleoTree package in R\\
packages used: Devtools, Claddis, PaleoTree \\

- What is the overall design of study?\\ %TG: uh? - OK I didn't check this bit, I assume they're just notes
- What are the variables and how do they relate to the hypotheses?\\
- How/where get the data?\\
- What are the characteristics of the data set / experiment -- how many observations, how many replicates etc.\\
- General procedures, if any, that are true in all of the analyses e.g., transformation of data, model checking, how models were compared\\
- How did you test the hypotheses, in the logical order outlined in the introduction i.e., from the general to the specific? Make sure you show that your tests are appropriate.\\
